operator/internal/status/lokistack.go

```diff
@@ -22,6 +19,7 @@ const (
 	messagePending                = "Some LokiStack components pending on dependencies"
 	messageDegradedMissingNodes   = "Cluster contains no nodes matching the labels used for zone-awareness"
 	messageDegradedEmptyNodeLabel = "No value for the labels used for zone-awareness"
+	messageOldSchemaVersion       = "The schema configuration contains one or more schemas that do not use the most recent version."
```

Comment:
I just realized, that this warning is still not actionable by the user. The "has future old versions" warning in the spike did go a step into the right direction but it also failed to make the point:

Currently the user can not remove old schema versions, so having a warning trigger on "there is an old schema version in the configuration" does not make sense, it only matters if the configuration ends with the latest version, so that the warning goes away once the v13 schema version is added.

So the following warning should make more sense:

```suggestion
	messageWarningNeedsSchemaVersionUpdate = "The schema configuration does not contain the most recent schema version and needs an update."
```

Reply:
This makes sense, thanks